RELEASE.md

```diff
@@ -60,52 +60,57 @@ Following Requirements needs to be met prior to final RC Cut:
 
 Release branches are typically cut from the branch [`viable/strict`](https://github.com/pytorch/pytorch/tree/viable/strict) as to ensure that tests are passing on the release branch.
 
-There's a convenience script to create release branches from current `viable/strict` (from root `pytorch/pytorch`):
+There's a convenience script to create release branches from current `viable/strict`. Perform following actions :
+* Perform a fresh clone of pytorch repo using
+```bash
+git clone git@github.com:pytorch/pytorch.git
+```
```

Comment:
This sounds like an optional step, isn't it? Also, why are you recommending using ssh rather than https access? Later is faster, isn't it?
```suggestion
* Perform a fresh clone of pytorch repo using
```bash
git clone https://github.com:pytorch/pytorch.git
```
```

Reply:
1. Basically this step is required if you want to execute next step which is convenience script, which should be executed from the pytorch clone. Executing the script from fork does not work. Hence fresh clone, which also makes sure no other accidental changes are picked up during release.

2.  I don't mind putting https here if it works well for next step, so far I have been always using ssh with keys setup for proper access. Main point here is that it must be pytorch/pytorch repo and not a fork.

Reply:
Sorry, I don't think I follow. What do you mean by:
> which should be executed from the pytorch clone
If script does not work from forks, then perhaps we need to add note about it here and also add a check for it in the script.

Reply:
Yes, makes sense we should probably change the script to work from forks properly.  The way I usually work is as follows:

```
(.venv) [atalman@devvm214.ftw0 ~/local/pytorch] git remote -v
origin  git@github.com:atalman/pytorch.git (fetch)
origin  git@github.com:atalman/pytorch.git (push)
upstream        git@github.com:pytorch/pytorch.git (fetch)
upstream        git@github.com:pytorch/pytorch.git (push)
```

The script by default assumes:  ```GIT_REMOTE=${GIT_REMOTE:-origin}``` hence if I execute it from fork it will push it to atalman/pytorch.

Perhaps we should just avoid using the script since it creates unnecessary confusion, all it does:
```
set -x
git checkout "${GIT_REMOTE}/${GIT_BRANCH_TO_CUT_FROM}"
git checkout -b "${branch}"
git push "${GIT_REMOTE}" "${branch}"
```

Maybe just executing these command against correct origin or upstream will be cleaner ?